gamemodes/terrortown/gamemode/client/cl_vskin/default_skin.lua

```diff
@@ -1205,5 +1211,81 @@ function SKIN:PaintEventBoxTTT2(panel, w, h)
 	end
 end
 
+---
+-- @param Panel panel
+-- @param number w
+-- @param number h
+-- @realm client
+function SKIN:PaintCardTTT2(panel, w, h)
+	local widthBorder = 2
+	local widthBorder2 = widthBorder * 2
+	local sizeIcon = 64
+	local padding = 5
+	local posIcon = widthBorder + padding
+	local posText = posIcon + sizeIcon + 2 * padding
+	local heightMode = 35
+	local widthMode = w - sizeIcon - 3 * padding
+	local posIconModeX = w - widthMode + 2 * padding
+	local posIconModeY = h - heightMode + 2 * padding
+	local sizeIconMode = heightMode - 4 * padding
+	local posTextModeX = posIconModeX + sizeIconMode + 2 * padding
+	local posTextModeY = posIconModeY + 0.5 * sizeIconMode - 1
```

Comment:
some of them can be localized outside this function. Same as some calculations ^^ It isn't a lot, but it's a paint method (called every frame) for each "object" of this Panel.

Reply:
Makes sense, but this is probably something Tim should do for all Skins. He uses this pattern for many other paint-functions.
In general showed my benchmark, that UI could need some optimization as it takes the most time at the moment.